pygbm/grower.py

```diff
@@ -413,25 +413,30 @@ def _finalize_splittable_nodes(self):
             node = self.splittable_nodes.pop()
             self._finalize_leaf(node)
 
-    def make_predictor(self, bin_thresholds=None):
+    def make_predictor(self, numerical_thresholds=None):
         """Make a TreePredictor object out of the current tree.
 
         Parameters
         ----------
-        bin_thresholds : array-like of floats, optional (default=None)
-            The actual thresholds values of each bin.
+        numerical_thresholds : array-like of floats, optional (default=None)
+            The actual thresholds values of each bin. None if the training data
+            was pre-binned.
```

Comment:
Maybe add that the values of the array are expected to be sorted in increasing order.